### PR TITLE
Align tooling with behance standards

### PR DESCRIPTION
**Pull request by weikinhuang**

### CODE REVIEW

File: karma-common-config.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* jshint node: true */
```
Comment (kaicataldo) [inline karma-common-config.js]: Can remove this line now

**Comment by kaicataldo** [comment]
I was wondering why jshint wasn't being removed from the package.json, haha

**Comment by kaicataldo** [comment]
Aside from comment above,

**Comment by kaicataldo** [comment]
:+1:

**Comment by weikinhuang** [comment]
jshint wasn't in there

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "start": "gulp",
-    "test": "gulp test"
+    "test": "eslint . && gulp test",
+    "lint": "eslint ."
```
Comment (hzoo) [inline package.json]: I guess `.` used to be slow so this should be ok

File: test/main.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
 /* global require */
 // Unbreak PhantomJS's Function.prototype.bind
 var isFunction = function(o) {
-  return typeof o == 'function';
+  return typeof o === 'function';
```
Comment (hzoo) [inline test/main.js]: weird that it's a bind polyfill and I don't see phantomjs in the deps

**Comment by hzoo** [comment]
👍

**Pull request merged**